app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -17,8 +17,8 @@ class Vmware::InfraManager < InfraManager
     require_nested :Provision
     require_nested :ProvisionViaPxe
     require_nested :ProvisionWorkflow
-    require_nested :Refresher
     require_nested :RefreshParser
+    require_nested :Refresher
```

Comment:
I vote to add a comment here in case someone tries that again in the future. :)

Reply:
Yes absolutely I wanted to make sure this was the only issue first.  I don't want people putting requires in random places again.